opensearch/src/main/java/org/opensearch/sql/opensearch/data/value/OpenSearchExprValueFactory.java

```diff
@@ -363,6 +363,49 @@ private ExprValue parseArray(
     return new ExprCollectionValue(result);
   }
 
+  /**
+   * Parse geo point content.
+   *
+   * @param content Content to parse.
+   * @param supportArrays Parsing the whole array or not
+   * @return Geo point value parsed from content.
+   */
+  private ExprValue parseGeoPoint(Content content, boolean supportArrays) {
```

Comment:
From the doc, there seems 6 ways to represent a geo point. Could you clarify in comment which code block is handling which format?